lighting/common/attenuation.glsl

```diff
@@ -0,0 +1,34 @@
+/*
+contributors: Shadi El Hajj
+description: Light attenuation equation
+license: MIT License (MIT) Copyright (c) 2024 Shadi EL Hajj
+*/
+
+#ifndef LIGHT_ATTENUATION_CONSTANT
+#define LIGHT_ATTENUATION_CONSTANT 0
+#endif
+
+#ifndef LIGHT_ATTENUATION_LINEAR
+#define LIGHT_ATTENUATION_LINEAR 0
+#endif
+
+#ifndef LIGHT_ATTENUATION_EXPONENTIAL
+#define LIGHT_ATTENUATION_EXPONENTIAL 1
+#endif
+
+#ifndef LIGHT_ATTENUATION_EXPONENT
+#define LIGHT_ATTENUATION_EXPONENT 2
+#endif
+
+#ifndef FNC_LIGHT_ATTENUATION
+#define FNC_LIGHT_ATTENUATION
+
+float attenuation(float dist) {
```

Comment:
I think this two files (`attenuation.glsl` and `attenuation.hlsl`) should be moved from `lighting/common/` to `lighting/light/`. Following the logic of https://github.com/patriciogonzalezvivo/lygia/blob/main/lighting/light/falloff.glsl

Reply:
Fixed!